releasenotes/notes/improve-systemd-config-a82bca0014f6ccc6.yaml

```
@@ -0,0 +1,20 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+upgrade:
+  - |
+    Update systemd configuration. Before if `unit_names` and `unit_regexes`
```

It's important to mention in the release notes here that they apply to the systemd _check_ specifically. Systemd in the context of the Agent can mean other things, for example how the Agent is started by systemd on some systems.

```suggestion
    Update to the configuration of the systemd check: Before, if `unit_names` and `unit_regexes`
```